packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.js

```diff
@@ -198,6 +198,7 @@ async function serviceWalkthrough(context, defaultValuesFilename, stringMapsFile
     ...coreAnswers,
     userPoolGroupList,
     adminQueryGroup,
+    serviceName: 'Cognito',
```

Comment:
Isn't the `serviceName` different for Admin Queries? IIRC the service name is used to persist admin queries in the state files.

Reply:
There are separate functions that generate the Admin Queries API that have separate hardcoded service names